### PR TITLE
Create antmedia-samples.py

### PR DESCRIPTION
**Pull request by yashtandon113**
https://github.com/ant-media/Ant-Media-Server/issues/4787

### CODE REVIEW

**Comment by mekya** [comment]
Hey @yashtandon113 , could you please check the sonatype-lift issue?

**Comment by yashtandon113** [comment]
Hi @mekya 
Can we force the sonatype-lift test for this script? According to my understanding, it considers the input URL (RTMP and SRT) to be untrusted. I attempted to secure the URL using different python packages, but it is still not passing.

Do you have any suggestions?

File: Selenium/antmedia-samples.py [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+        print("Slack message sent successfully!")
+
+#Function to start FFMPEG process
+def publish_with_ffmpeg(url, protocol='rtmp'):
+    if protocol == 'rtmp':
+        # Start FFmpeg process for RTMP streaming
+        quoted_url = shlex.quote(url)
+        ffmpeg_command = 'ffmpeg -re -f lavfi -i smptebars -c:v libx264 -preset veryfast -tune zerolatency -profile:v baseline -c:a aac -b:a 128k -t 30 -f flv' + ' ' + quoted_url
+        ffmpeg_args = shlex.split(ffmpeg_command)
+        ffmpeg_process = subprocess.Popen(ffmpeg_args, stdout=subprocess.PIPE, stderr=subprocess.PIPE, shell=False)
```
[the remark this answers is not in the document]
Comment (yashtandon113) [reply Selenium/antmedia-samples.py]: @sonatype-lift ignore

File: Selenium/antmedia-samples.py [hunk 2]
```diff
@@ -0,0 +1,279 @@
+import os
+import time
+import json
+import shlex
+import requests
+import subprocess
+import urllib.parse
+from selenium import webdriver
```
[the remark this answers is not in the document]
Comment (yashtandon113) [reply Selenium/antmedia-samples.py:8]: @sonatype-lift ignoreall

File: Selenium/antmedia-samples.py [hunk 3]
```diff
@@ -0,0 +1,283 @@
...
+for i in range(2):
+    try:
+        driver.execute_script("window.open('https://antmedia.io/webrtc-samples/webrtc-virtual-background/', '_blank');")
+        driver.switch_to.window(driver.window_handles[1])
+        time.sleep(2)
+        remove_ad(driver)
+        time.sleep(20)
+        driver.switch_to.frame(0)
+        time.sleep(3)
+        driver.find_element(By.XPATH, "/html/body/div/div/div[4]/div[3]/img").click()
```
[the remark this answers is not in the document]
Comment (yashtandon113) [reply Selenium/antmedia-samples.py:98]: @sonatype-lift ignoreall

File: Selenium/antmedia-samples.py [hunk 4]
```diff
@@ -0,0 +1,299 @@
...
+switch_to_first_tab(driver)
+
+# Testing WebRTC and HLS Comparison Live Demo Page
+try:
+    driver.execute_script("window.open('https://antmedia.io/live-demo/', '_blank');")
+    driver.switch_to.window(driver.window_handles[1])
+    time.sleep(20)
+    remove_ad(driver)
+    time.sleep(2)
+    driver.find_element(By.XPATH, "/html/body/div[5]/div/article[2]/div[2]/div[1]/div[1]/div/div/p/button[1]").click()
```
[the remark this answers is not in the document]
Comment (yashtandon113) [reply Selenium/antmedia-samples.py:120]: @sonatype-lift ignoreall

File: Selenium/antmedia-samples.py [hunk 5]
```diff
@@ -0,0 +1,311 @@
...
+    print("WebRTC Deep AR test is successful")
+
+except:
+    message = "WebRTC DeepAR sample test is failed, check -> https://antmedia.io/webrtc-samples/deepar-publish-play/"
+    send_slack_message(webhook_url, message, icon_emoji)
+
+switch_to_first_tab(driver)
+
+try:
+    driver.execute_script("window.open('https://antmedia.io/webrtc-samples/interactive-whiteboard-publish-play/', '_blank');")
```
[the remark this answers is not in the document]
Comment (yashtandon113) [reply Selenium/antmedia-samples.py:271]: @sonatype-lift ignoreall

**Review by muratugureminoglu — APPROVED** [review]

**Pull request merged**